common/djangoapps/util/markup.py

```diff
@@ -0,0 +1,51 @@
+"""
+Utilities for use in Mako markup.
+"""
+
+from django.utils.translation import ugettext as django_ugettext
+from django.utils.translation import ungettext as django_ungettext
+import markupsafe
+
+
+# So that we can use escape() imported from here.
+escape = markupsafe.escape                      # pylint: disable=invalid-name
+
+
+def ugettext(text):
+    """Translate a string, and escape it as plain text.
+
+    Use like this in Mako::
+
+        <% from util.markup import ugettext as _ %>
+        <p>${_("Hello, world!")}</p>
+
+    Or with formatting::
+
+        <% from util.markup import HTML, ugettext as _ %>
+        ${_("Write & send {start}email{end}").format(
+                start=HTML("<a href='mailto:ned@edx.org'>"),
+                end=HTML("</a>"),
+                )}
```

Comment:
I think sample code still requires "| h" filter for now because we don't yet have this as a default.